src/Compilation.zig

```diff
@@ -348,6 +348,8 @@ pub const InitOptions = struct {
     want_valgrind: ?bool = null,
     use_llvm: ?bool = null,
     use_lld: ?bool = null,
+    /// When set, this option will overwrite LLD with the system linker LD.
```

Comment:
```suggestion
    /// When set, this option will invoke the system linker ld rather than using LLD
    /// Only allowed when compiling natively for macos, and this option will be removed in the future,
    /// after the LLD codebase is improved.
```